source/mod_boinc.f90

```diff
@@ -151,21 +161,27 @@ subroutine boinc_post
 end subroutine boinc_post
 
 ! ================================================================================================ !
-!  Report exit status, close the log file, and tell BOINC to finish
+!  Bump progress to 100%, and close the log file
 ! ================================================================================================ !
-subroutine boinc_finalise(exitCode)
+subroutine boinc_done
 
   use mod_units
 
-  integer, intent(in) :: exitCode
+#ifdef API
+  call boinc_fraction_done(1.0)
+#endif
+  write(boinc_logBuffer,"(a,f8.3,a)") "Progress: ",100.0," %"
```

Comment:
following comment above, don't you want to report, instead of 100%, a statement like:"call to boinc done"?

Reply:
No, this is better I think as it actually does reach 100% in the log file. Otherwise the progress counter may be left hanging on 99%.